test/benchmark-common-tasks/main.cpp

```diff
@@ -1630,6 +1630,72 @@ struct BenchmarkSort : BenchmarkWithStrings {
     }
 };
 
+// benchmark for testing compressed string sorting.
+// N is the size of the string to generate
+// M is the numer of times we want store the string (number of dups)
+template <size_t N, size_t M>
+struct BenchmarkSortCompressed : BenchmarkWithStrings {
+    mutable std::string _name;
+
+    BenchmarkSortCompressed()
+        : BenchmarkWithStrings()
+    {
+        if constexpr (N <= 15) {
+            _name = util::format("SortCompressedSmall(%1,%2)", std::to_string(N), std::to_string(M));
```

Comment:
is the `std::to_string(N)` necessary? I would expect `util::format` to do the right thing with a constant number.